src/main/java/com/misterveiga/cds/listeners/MessageListener.java

```diff
@@ -580,4 +586,80 @@ private EmbedBuilder getRobloxUserInfoEmbed(String RobloxUserName, String UserId
 			return embed;
 		}
 	}
+	
+	
+	/** The virus total token. */
+	@Value("${virus.total.token}")
+	public String virusTotalToken;
+	
+	private EmbedBuilder scanUrl(String url, String botIcon) {
```

Comment:
Embed builders should be in their own class OR VirusTotal implementation should be in another class to avoid cluttering Message Listener (invoking from the listener is OK).